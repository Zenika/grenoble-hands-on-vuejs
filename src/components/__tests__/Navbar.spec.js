import { shallowMount } from '@vue/test-utils'
import Navbar from '@/components/Navbar.vue'

describe('Footer.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Navbar)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
