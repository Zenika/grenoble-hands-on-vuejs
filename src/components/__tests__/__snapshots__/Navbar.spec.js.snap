// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue renders correctly 1`] = `
<nav class="navbar is-primary" role="navigation" aria-label="main navigation">
  <div class="container">
    <div class="navbar-brand">
      <h1 class="title navbar-item"> Zenika Cities weather </h1>
    </div>
    <div class="navbar-menu">
      <h2 class="navbar-item has-text-white"> The best weather app in Vue </h2>
    </div>
    <div class="navbar-menu navbar-end">
      <router-link class="navbar-item" to="[object Object]"> Create city </router-link>
    </div>
  </div>
</nav>
`;
